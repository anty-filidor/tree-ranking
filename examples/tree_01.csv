id;parent_id;color;value;​
1;;yellow;70;​
2;1;blue;20;​
3;1;yellow;30;​
4;2;blue;80;​
5;3;yellow;200;​
6;3;green;400;​
7;6;blue;200;
