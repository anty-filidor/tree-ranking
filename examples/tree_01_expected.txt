python color_ranking.py --file /path/tree_01.csv --criteria subtree-count​
[('blue', 2), ('green', 1), ('yellow', 1)]​

​
python color_ranking.py --file /path/tree_01.csv --criteria subtree-maxdepth​
[('yellow', 3), ('blue', 2), ('green', 1)]​

​
python color_ranking.py --file /path/tree_01.csv --criteria subtree-average-value​
[('green', 400), ('yellow', 300), ('blue', 150)]​
